### PR TITLE
Bug/735  initial page load or refresh shows no users available in as…

### PR DESCRIPTION
**Pull request by jananikuruntham**
…signed users issues has been fixed

### CODE REVIEW

File: api/src/group-users/group-users.service.ts [hunk 1]
```diff
@@ -135,18 +135,23 @@ export class GroupUsersService {
...
       where: whereCondition,
     });
 
-    if (groupUsers.length === 0) {
-      return {
-        message: 'Group user not found.',
-        statusCode: HttpStatus.OK,
-      };
-    }
+    console.log('gh', groupId);
```
Comment (vsaravanakumar) [inline api/src/group-users/group-users.service.ts:154]: Remove console.log

File: front/src/app/groups/[id]/page.tsx [hunk 2]
```diff
@@ -743,15 +758,21 @@ const GroupView = ({ params }: { params: IGroupView }) => {
                                   />
                                 ))}
 
+                              {/* {options.filter((option) =>
```
Comment (vsaravanakumar) [inline front/src/app/groups/[id]/page.tsx:761]: Remove commented lines

File: front/src/components/Profile/Profile.tsx [hunk 3]
```diff
@@ -277,18 +278,24 @@ const ProfilePage = () => {
...
       try {
         const updatedData = {
           ...editedData,
           id: id,
-          currentPassword: currentPassword,
+          currentPassword: verifyCurrentPassword,
         };
-        const response = await UserApi.verifyCurrentPassword(id, updatedData);
+        const response = await UserApi.checkCurrentPassword(id, updatedData);
+        console.log("pp",response);
```
Comment (vsaravanakumar) [inline front/src/components/Profile/Profile.tsx:290]: Remove console.log

File: api/src/group-users/group-users.service.ts (hunk 1 above)
Comment (jananikuruntham) [reply api/src/group-users/group-users.service.ts]: removed console log

File: front/src/app/groups/[id]/page.tsx (hunk 2 above)
Comment (jananikuruntham) [reply front/src/app/groups/[id]/page.tsx]: removed commented lines

File: front/src/components/Profile/Profile.tsx (hunk 3 above)
Comment (jananikuruntham) [reply front/src/components/Profile/Profile.tsx]: removed console log

**Pull request merged**